meta-balena-common/classes/kernel-balena.bbclass

```diff
@@ -247,6 +247,7 @@ BALENA_CONFIGS[nfsfs] = " \
     CONFIG_NFS_V2=m \
     CONFIG_NFS_V3=m \
     CONFIG_NFS_V4=m \
+    CONFIG_NFSD=m \
```

Comment:
Should this merge before https://github.com/balena-os/balena-raspberrypi/pull/1104?

Reply:
Indeed, thanks.